tests/unit-tests/test-class-sensei-context-notices.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/**
+ * Tests for Sensei_Context_Notices class.
+ *
+ * @group context-notices
+ */
+class Sensei_Context_Notices_Test extends WP_UnitTestCase {
+	/**
+	 * Set up the test.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->factory = new Sensei_Factory();
```

Comment:
What is this used for?

Reply:
Ah, it's not necessary here! Removed in: https://github.com/Automattic/sensei/pull/4477/commits/200a8f685689e6b28792e527faa806147b648ae3

I started writing it as part of a block for the post types, and after changed it, I forgot to remove this. Good catch!